.env.example

```diff
@@ -8,6 +8,7 @@ IMAGES_AWS_BUCKET=coordinape
 IMAGE_DIR=assets/static/images/
 # separate allowed domains with commas
 SIWE_ALLOWED_DOMAINS=localhost:3000,
+DEV_SEED_PHRASE="test test test test test test test test test test test junk"
```

Comment:
Seems to not work... and the process.env.DEV_SEED_PHRASE just seems to interpret this string as "test" with only one word... unsure hwo to pass a string with spaces here?

Reply:
Spaces are supposed to work: https://github.com/motdotla/dotenv#what-rules-does-the-parsing-engine-follow